cli/lib/args.rs

```diff
@@ -89,7 +91,9 @@ pub fn get_root_cert_store(
         root_cert_store.extend(webpki_roots::TLS_SERVER_ROOTS.to_vec());
       }
       "system" => {
-        let roots = load_native_certs().expect("could not load platform certs");
+        let roots = load_native_certs().map_err(|err| {
+          RootCertStoreLoadError::FailedNativeCerts(err.to_string())
```

Comment:
Maybe change to:

```
FailedNativeCerts {
  #[source]
  source: ...
}

Reply:
Cannot store `std::io::Error` because `RootCertStoreLoadError` needs to implement `Clone`